Etabs_Adapter/ETABSAdapter.cs

```diff
@@ -138,6 +140,8 @@ public ETABSAdapter(string filePath = "", EtabsSettings etabsSetting = null, boo
                         m_model.File.NewBlank();
                 }
 
+                FilePath = m_model.GetModelFilename();
```

Comment:
```suggestion
                FilePath = m_model.GetModelFilename();
```

Has this been tested after using the `Execute` action to create a new file - it's common that they won't have a filename until you SaveAs - what happens then?

Reply:
Hi @peterjamesnugent, thanks a lot for this comment.
I've tested the code with Execute and added a few lines of code that allow to update the FilePath as soon as the new un-named model gets saved.
When the model is unsaved, the FilePath returns a "(Untitled)" string to signify that the model has no path currently.
All saved in latest commit ;)
you can find updated test GH script at same location (see below)
https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/ETABS_Toolkit/%23478-GetEtabsFilePath/Test%20Script.gh?csf=1&web=1&e=dEf085